packages/node/src/index.ts

```diff
@@ -97,3 +97,10 @@ const INTEGRATIONS = {
 export { INTEGRATIONS as Integrations, Handlers };
 
 export { hapiErrorPlugin } from './integrations/hapi';
+
+import { instrumentNodeCron } from './cron/node-cron';
+
+/** Methods to instrument cron libraries for Sentry check-ins */
+export const cron = {
```

Comment:
we might also need to re-export these from other packages that re-export node stuff (serverless, remix, sveltekit)